### ISSUE TITLE
Favorite/Starred Users

### ISSUE THREAD

**Issue by jb55** [damus-io/damus#2438]
As a Damus user who wants to have a better list experience, I would like to favourite or "star" certain users, so that I can have an easy first "Favorite follow list".

**Is your feature request related to a problem? Please describe.**

A common problem is that the people who post the most dominate the feed, while interesting people who don't post very often.

We are planning user lists:

- #531 

But this would be a very simple implementation of a list. You simply star a user to add them to your favorites. This maintains a favorite user list which is easier to get into without having to create a new list and add users to that list manually.

**Describe alternatives you've considered**

Alternatively we can build the full list UI and just use that like Xitter, but favourites would cover 90% of what most people want. I would suspect most people aren't creating many lists, they really just want a single favourite list of people they want to follow more closely.

**Comment by alltheseas** [comment]
Is it a private list? 
Is it nip-51 compatible?

**Comment by jb55** [comment]
@alltheseas all lists are nip-51 compatible, there is no reason to do something other than nip-51

If we make them private by default, then we won't be able to do:

- #2441 

unless we make it a private thing on the purple api.

**Comment by alltheseas** [comment]
Related: receive notifications from people I subscribe to

https://github.com/damus-io/damus/issues/1265

**Comment by danieldaquino** [comment]
@alltheseas @robagreda, although I imagine the UI/UX for this could be as simple as adding a star/heart button somewhere on the user profiles, I marked this one with a "design" label in case you guys have any cool ideas on how to design this special list

**Comment by robagreda** [comment]
Yeah, I am on this! hehe taking time because I am designing things for notedeck but once I am ready I will tackle this! Thanks!

**Comment by alltheseas** [comment]
Thanks Rob, looking forward to your masterpiece :eyes: 

> although I imagine the UI/UX for this could be as simple as adding a star/heart button somewhere on the user profiles

love the simplicity of this approach Daniel

**Comment by alltheseas** [comment]
@danieldaquino using heart, instead of star, as not to confuse with purple subscription

**Comment by alltheseas** [comment]
deferring post video fixes, and tap sensitivity fixes

**Comment by alltheseas** [comment]
http://damus.io/figma

**Comment by alltheseas** [comment]
@askeew I updated the ticket with acceptance criteria, and a link to the design system.

@jb55 @danieldaquino let me know if acceptance criteria are sufficient/too much/insufficient 🙏

**Comment by danieldaquino** [comment]
> let me know if acceptance criteria are sufficient/too much/insufficient 🙏

Looks good to me, thank you @alltheseas!
